A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AirspeedIndicator.js

```diff
@@ -1810,16 +1811,22 @@ class Jet_PFD_AirspeedIndicator extends HTMLElement {
             this._lastIASTime = newIASTime;
             return;
         }
+        
         let frameIASAcceleration = (newIASTime.ias - this._lastIASTime.ias) / (newIASTime.t - this._lastIASTime.t);
+        
         frameIASAcceleration = Math.min(frameIASAcceleration, 10);
         frameIASAcceleration = Math.max(frameIASAcceleration, -10);
+        
         if (isFinite(frameIASAcceleration)) {
-            this._computedIASAcceleration *= 0.998;
-            this._computedIASAcceleration += frameIASAcceleration * 0.002;
+            // Low pass filter for accel : https://en.wikipedia.org/wiki/Low-pass_filter
+            this._computedIASAcceleration = this._computedIASAcceleration
+                                          + this._accelAlpha * (frameIASAcceleration - this._computedIASAcceleration);
         }
+
         this._lastIASTime = newIASTime;
-        let accel = this._computedIASAcceleration * 10;
-        return accel;
+
+        let accel = this._computedIASAcceleration;
+        return accel * 10;
```

Comment:
Why not just:
```suggestion
        return this._computedIASAcceleration * 10;
```

Reply:
I was logging it - I can simplify it though.